### PR TITLE
A bunch of tested proton games

### PR DESCRIPTION
**Pull request by daddeltrotter**

### CODE REVIEW

**Comment by alkazar** [comment]
@daddeltrotter Sorry for the delay. It seems there are some conflicts, could you please merge in latest master and resolve them? Thank you!

**Comment by daddeltrotter** [comment]
@alkazar: I've resolved the conflicts. Hope everything is fine now.

File: steam-tweaks.yaml [hunk 1]
```diff
@@ -402,6 +447,15 @@
         src: data/Freedom Planet/control_gamepad.cfg
         dst: control_gamepad.cfg
 
+"1047220": # Frog Detective 2 The_Case of the Invisible Wizard
```
Comment (alkazar) [inline steam-tweaks.yaml:450]: title should be: "Frog Detective 2: The Case of the Invisible Wizard"

File: steam-tweaks.yaml [hunk 2]
```diff
@@ -382,12 +415,24 @@
...
 "919410": # Felix the Reaper
   compat_tool: proton_411
 
 "637650": # FINAL FANTASY XV WINDOWS EDITION
   compat_tool: proton_513
 
+"318600": # The Flame in the Flood
+  compat_tool: proton_513
+
+"405970": # Flat Kingdom Papers Cut Edition
```
Comment (alkazar) [inline steam-tweaks.yaml:433]: title should be: "Flat Kingdom Paper's Cut Edition"

File: steam-tweaks.yaml [hunk 3]
```diff
@@ -402,6 +447,15 @@
         src: data/Freedom Planet/control_gamepad.cfg
         dst: control_gamepad.cfg
 
+"1047220": # Frog Detective 2 The_Case of the Invisible Wizard
+  compat_tool: Proton-5.21-GE-1
+
+"1142500": # Fun with Ragdolls The Game
```
Comment (alkazar) [inline steam-tweaks.yaml:453]: title should be: "Fun with Ragdolls: The Game"

File: steam-tweaks.yaml [hunk 4]
```diff
@@ -801,20 +873,26 @@
...
+  compat_tool: proton_513
+
 "287260": # Toybox Turbos
   compat_tool: proton_513
   compat_config: noesync
 
 "585420": # Trailmakers
   compat_tool: proton_42
 
+"914010": # TrainStation Renovation
```
Comment (alkazar) [inline steam-tweaks.yaml:886]: title should be: "Train Station Renovation"

File: steam-tweaks.yaml [hunk 5]
```diff
@@ -851,15 +929,16 @@
   compat_tool: proton_411
   launch_options: --skip-intro
 
+"774241": # Warhammer Chaosbane
```
Comment (alkazar) [inline steam-tweaks.yaml:932]: title should be: "Warhammer: Chaosbane"

File: steam-tweaks.yaml [hunk 6]
```diff
@@ -801,20 +873,26 @@
...
+  compat_tool: Proton-5.21-GE-1
+ 
 "924980": # Trials of Mana
   compat_tool: Proton-5.21-GE-1
 
 "319910": # Trine 3: The Artifacts of Power; native version shows generic gamepad hints
   compat_tool: proton_42
 
-"319910": # Trine 4: The Nightmare Prince
+"690640": # Trine 4: The Nightmare Prince
```
Comment (alkazar) [inline steam-tweaks.yaml:895]: Oops, I must have copied the Trine 3 app id. Nice catch!

File: steam-tweaks.yaml [hunk 7]
```diff
@@ -851,15 +929,16 @@
   compat_tool: proton_411
   launch_options: --skip-intro
 
+"774241": # Warhammer Chaosbane
+  compat_tool: proton_513
+  steam-input: enabled
```
Comment (alkazar) [inline steam-tweaks.yaml:934]: It should actually be "steam_input", not "steam-input". But, "steam_input: enabled" has no effect on games run with proton. Steam Input is always enabled for Proton games. So this line should be removed.

File: steam-tweaks.yaml [hunk 8]
```diff
@@ -851,15 +929,16 @@
...
+  
 "865360": # We Were Here Together
   compat_tool: proton_411
 
 "526160": # The Wild Eight
   compat_tool: proton_5
 
-"1142100": # WWE 2K BATTLEGROUNDS
```
Comment (alkazar) [inline steam-tweaks.yaml:860]: Oops, my bad.

File: steam-tweaks.yaml [hunk 9]
```diff
@@ -887,6 +966,9 @@
 "354830": # Wolfenstein: The Old Blood German Edition
   compat_tool: proton_513
 
+"1142100": # WWE 2K BATTLEGROUNDS
```
Comment (alkazar) [inline steam-tweaks.yaml:969]: Thanks!

**Review by alkazar — CHANGES_REQUESTED** [review]

File: steam-tweaks.yaml (hunk 2 above)
Comment (daddeltrotter) [reply steam-tweaks.yaml]: Thanks. Changed it.

File: steam-tweaks.yaml (hunk 1 above)
Comment (daddeltrotter) [reply steam-tweaks.yaml]: Thanks changed it.

File: steam-tweaks.yaml (hunk 3 above)
Comment (daddeltrotter) [reply steam-tweaks.yaml]: Thanks. Changed it.

File: steam-tweaks.yaml (hunk 4 above)
Comment (daddeltrotter) [reply steam-tweaks.yaml]: Thanks. Changed it.

File: steam-tweaks.yaml (hunk 5 above)
Comment (daddeltrotter) [reply steam-tweaks.yaml]: Thanks. changed it.

File: steam-tweaks.yaml (hunk 7 above)
Comment (daddeltrotter) [reply steam-tweaks.yaml]: I will remove it. 
But I'm pretty sure that only with Steam Input enabled the controller was detected. 
I found similar case with "NiGHTS Into Dreams".
I will check again the two games.

**Pull request merged**